### ISSUE TITLE
WatchConnectionManager warning log provides little information

### ISSUE THREAD

**Issue by shawkins** [fabric8io/kubernetes-client#3001]
We see occurrences of the following:

```
2021-04-14 03:49:17,302 WARN  [io.fab.kub.cli.dsl.int.WatchConnectionManager] (OkHttp https://172.30.0.1/...) Exec Failure: java.io.EOFException
	at okio.RealBufferedSource.require(RealBufferedSource.java:65)
	at okio.RealBufferedSource.readByte(RealBufferedSource.java:78)
	at okhttp3.internal.ws.WebSocketReader.readHeader(WebSocketReader.java:117)
	at okhttp3.internal.ws.WebSocketReader.processNextFrame(WebSocketReader.java:101)
	at okhttp3.internal.ws.RealWebSocket.loopReader(RealWebSocket.java:273)
	at okhttp3.internal.ws.RealWebSocket$1.onResponse(RealWebSocket.java:209)
	at okhttp3.RealCall$AsyncCall.execute(RealCall.java:174)
	at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

But that conveys very little information.  Presumably this can happen if the other side drops the websocket connection.  Could the message be improved or the stacktrace omitted for EOFExceptions?

**Comment by manusa** [comment]
Relates to #2993

**Comment by shawkins** [comment]
Closed via #3197
